commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/json/serialization/SectionJsonSerializer.kt

```diff
@@ -8,9 +8,11 @@ import com.fasterxml.jackson.databind.SerializerProvider
 import com.fasterxml.jackson.databind.ser.std.StdSerializer
 import ebi.ac.uk.model.Section
 import ebi.ac.uk.model.constants.SectionFields
+import ebi.ac.uk.model.extensions.fileListName
 
 internal class SectionJsonSerializer : StdSerializer<Section>(Section::class.java) {
     override fun serialize(section: Section, gen: JsonGenerator, provider: SerializerProvider) {
+        addFileListExt(section)
```

Comment:
mutate an object as part of serialization is an unexpected/unwanted side effect , we can just serialize as expected